packages/core/src/components/Toggle/MockToggle.tsx

```diff
@@ -22,15 +24,17 @@ export const MockToggle: FC<MockToggleProps> = ({
   onOverrideText,
   className,
   selectedClassName,
-  disabled
+  disabled,
+  size = "medium"
 }) => (
   <>
     {areLabelsHidden ? null : <ToggleText disabled={disabled}>{offOverrideText}</ToggleText>}
     <div
       className={cx(styles.toggle, className, {
         [cx(styles.selected, selectedClassName)]: checked,
         [styles.notSelected]: !checked,
-        [styles.disabled]: disabled
+        [styles.disabled]: disabled,
+        [styles.small]: size === "small"
```

Comment:
I believe a better practice here can be to set a className with the size (whether it is a small/medium, e.g. `getStyle(styles, size)`), and afterwards change the css by only a css var.

e.g. instead of declaring width and height twice, declare the width and height to be a --toggle-width for example, with .small class to change the value of the css var

this can also help understand the calculation being done on the css

Reply:
this is not a blocker by me, just for your consideration

Reply:
@YossiSaadi Good idea, should I declare --toggle-width, --toggle-height, in a global file or in this one?

Reply:
it should be in the same module of MockToggle so different toggles can have different values for the vars
so for example
```css
.mockToggle {
  width: var(--toggle-width);
  &.small {
    --toggle-width: X1px;
    --toggle-height: Y1px;
  }
  &.medium {
    --toggle-width: X2px;
    --toggle-height: Y2px;
  }
}
```

Reply:
Done